FirebaseDev.podspec

```diff
@@ -16,6 +16,7 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
   s.source           = { :git => 'https://github.com/firebase/firebase-ios-sdk.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/Firebase'
   s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.10'
```

Comment:
If you mean in order to publish early, then instead of duplicating this, disabling iOS, and publishing as FirebaseMac, how about just pushing this one as-is as FirebaseDev? Then people who are eager can test both sides, integrate in cross platform frameworks, etc.